### PR TITLE
feat: Make a generic glimmer component

### PR DESCRIPTION
**Pull request by darryltec**
## Motivations

This enables us to easily create a consistent skeleton UI. Other teams can use this as well if they do want to build their own right now. Eventually a more opinionated, ready-to-use, plug-and-play skeleton UI would exists.

### Added

- Glimmer component

### Changed

- Use glimmer in format file image loading

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

JOB-66764

### CODE REVIEW

File: packages/components/src/Glimmer/style/Shape.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.rectangle {
+  width: 100%;
+}
+
+.rectangleShort {
+  width: 85%;
+}
+.rectangleShorter {
+  width: 70%;
+}
```
Comment (darryltec) [inline packages/components/src/Glimmer/style/Shape.css:10]: Not really in love with how shorter glimmers can be defined tho so if you have another idea, let me hear it!

**Comment by taylorvnoj** [comment]
👀

File: packages/components/src/Glimmer/Glimmer.test.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  });
+
+  describe("Shape", () => {
+    it.each<[Shapes]>([
+      ["rectangle"],
+      ["rectangleShort"],
+      ["rectangleShorter"],
+      ["square"],
+      ["circle"],
+    ])("should render a $s", expectedShape => {
```
Comment (taylorvnoj) [inline packages/components/src/Glimmer/Glimmer.test.tsx:33]:
```suggestion
    ])("should render a %s", expectedShape => {
```

File: packages/components/src/Glimmer/Glimmer.test.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+  describe("Size", () => {
+    it.each<[Sizes]>([
+      ["small"],
+      ["base"],
+      ["large"],
+      ["larger"],
+      ["largest"],
+      ["auto"],
+    ])("should render as $s", expectedSize => {
```
Comment (taylorvnoj) [inline packages/components/src/Glimmer/Glimmer.test.tsx:48]:
```suggestion
    ])("should render as %s", expectedSize => {
```

File: packages/components/src/Glimmer/Glimmer.test.tsx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+      render(<Glimmer timing="fast" size={expectedSize} />);
+      const element = screen.getByTestId(GLIMMER_TEST_ID);
+      expect(element).toHaveClass(expectedSize);
+    });
+  });
+
+  describe("Timing", () => {
+    it.each<[Timings]>([["base"], ["fast"]])(
+      "should render as $s",
```
Comment (taylorvnoj) [inline packages/components/src/Glimmer/Glimmer.test.tsx:57]:
```suggestion
      "should render as %s",
```

File: packages/components/src/Glimmer/style/Sizes.css [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+.larger {
+  height: var(--space-larger);
+}
+
+.largest {
+  height: var(--space-largest);
+}
+
+.auto {
```
Comment (taylorvnoj) [inline packages/components/src/Glimmer/style/Sizes.css:21]: Just because I can already hear it being asked - why not all spacing sizes?

**Review by taylorvnoj — CHANGES_REQUESTED** [review]
Just a couple questions and small suggestions 🚀

File: packages/components/src/Glimmer/style/Sizes.css (hunk 5 above)
Comment (darryltec) [reply packages/components/src/Glimmer/style/Sizes.css:21]: and I would ask the question back, why should it?

**Review by taylorvnoj — APPROVED** [review]
LGTM once circle is 🟢

**Pull request merged**